appstudio-controller/controllers/appstudio.redhat.com/environment_controller.go

```diff
@@ -125,11 +127,14 @@ func (r *EnvironmentReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 		if apierr.IsNotFound(err) {
 			// B) The GitOpsDeploymentManagedEnvironment doesn't exist, so needs to be created.
-
-			log.Info("Creating GitOpsDeploymentManagedEnvironment", "managedEnv", desiredManagedEnv.Name)
 			if err := rClient.Create(ctx, desiredManagedEnv); err != nil {
 				return ctrl.Result{}, fmt.Errorf("unable to create new GitOpsDeploymentManagedEnvironment: %v", err)
 			}
+
+			log.Info("Created GitOpsDeploymentManagedEnvironment", "managedEnv", desiredManagedEnv.Name,
+				sharedutil.LogAction, sharedutil.LogAction_Add,
+				sharedutil.LogAudit, true)
+
```

Comment:
IMHO, if we are already calling `sharedutil.LogAPIResourceChangeEvent`, we probably can skip adding `LogAction` and `LogAudit` to this log statement above the call to `LogAPIResourceChangeEvent`. `LogAPIResourceChangeEvent` will already output this data. (Same with similar cases below)